Zuora 5.0 Documentation

valkyrie69 opened the pull request.

valkyrie69 committed:
Zuora 5.0 Documentation

valkyrie69 committed:
Added Upgrade Info

valkyrie69 committed:
Update zuora-connector-reference.adoc

kbostrom commented on modules/ROOT/nav.adoc:
```diff
@@ -370,3 +370,4 @@
  ** xref:zuora/zuora-connector-xml-maven.adoc[Zuora Connector XML and Maven Support]
```
Instructions for edit review:

In this one, you only need to edit the file named zuora-connector-upgrade-migrate.adoc. The other files are either generated or files from the template that you already edited. Although, of course if you see anything (other than in the files with "reference" in the name) feel free to point it out! If you could get this done by Monday, that would be great.

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+= Zuora - Upgrade and Migrate - Mule 4
```
EDIT THIS FILE

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+The main changes introduced in Zuora Connector 5.0.0 are:
+
+* Four operations were removed from the connector and were added separately in the new Anypoint Connector for AQuA (AQuA Connector). +
```
```suggestion
* Four operations were removed from the connector and were added separately in the new Anypoint Connector for AQuA (AQuA Connector): +
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+The main changes introduced in Zuora Connector 5.0.0 are:
+
+* Four operations were removed from the connector and were added separately in the new Anypoint Connector for AQuA (AQuA Connector). +
+The removed operations are:
```
```suggestion
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+=== CRUD Operations
+
+All of the CRUD operations were changed so that CRUD is prepended to operation names, for example: 
```
```suggestion
All of the CRUD operations were changed so that CRUD is prepended to operation names: 
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+=== Changes in Fields and Values
+
+There are some changes in fields and their corresponding values:
```
```suggestion
Some fields and their corresponding values have changed:
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+There are some changes in fields and their corresponding values:
+
+* The credit memo items operation and the taxation items created in the `write-off` operation are now applied to the corresponding invoice items and invoice taxation items respectively.
```
```suggestion
* The credit memo items operation and the taxation items created in the `write-off` operation are now applied to the corresponding invoice items and invoice taxation items, respectively.
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+Before you perform the upgrade, you must:
+
+. Create a backup of your files, data, configuration in case you need to restore to the previous version. 
```
```suggestion
. Create a backup of your files, data, and configuration in case you need to restore to the previous version. 
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+. Create a backup of your files, data, configuration in case you need to restore to the previous version. 
+. Install Zuora AQuA Connector to replace the AQuA operations that were previously included in Zuora Connector 4.2.x:
+.. In Studio, create a Mule project.
+.. In the Mule Palette, click *Search in Exchange*.
```
```suggestion
.. In the Mule Palette view, click *Search in Exchange*.
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+Follow these steps to perform the upgrade to Zuora Connector 5.0.0:
+
+.. In Studio, create a Mule project.
+.. In the Mule Palette, click *Search in Exchange*.
```
```suggestion
.. In the Mule Palette view, click *Search in Exchange*.
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+After you install the latest version of the connector, follow these steps to verify the upgrade:
+
+. In Anypoint Studio, verify there are no errors in the *Problems* or *Console* views.
```
```suggestion
. In Anypoint Studio, verify that there are no errors in the *Problems* or *Console* views.
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+After you install the latest version of the connector, follow these steps to verify the upgrade:
+
+. In Anypoint Studio, verify there are no errors in the *Problems* or *Console* views.
+. Check the project pom.xml and verify there are no problems.
```
```suggestion
. Check the project pom.xml and verify that there are no problems.
```

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+. In Anypoint Studio, verify there are no errors in the *Problems* or *Console* views.
+. Check the project pom.xml and verify there are no problems.
+. Test the connection and verify the operations work.  
```
```suggestion
. Test the connection and verify that the operations work.  
```

kbostrom replied:
RE: use of "verify" vs. "verify that":

When you use the verb "verify," it is normally followed by a noun or a "that"-clause. 
A that-clause would normally contain a verb, like "solve" in the following example:

We need to verify the new version. 
We need to verify that the new version of the library solves the following issues list.

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+If there are problems with caching the parameters and caching the metadata, try restarting Anypoint Studio.  
+
+=== Reverting the Upgrade
```
Is this supposed to be a subtopic to Troubleshooting?

kbostrom commented on modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc:
```diff
@@ -0,0 +1,152 @@
+
+=== Reverting the Upgrade
+
+If it is necessary to revert back to the previous version of Zuora Connector, change the `zuora-connector` dependency version `5.0.0` in the project's pom.xml to the previous version.  
```
```suggestion
If it is necessary to revert to the previous version of Zuora Connector, change the `zuora-connector` dependency version `5.0.0` in the project's pom.xml to the previous version.  
```

kbostrom reviewed (approved):
Here you go!

valkyrie69 committed:
Merge branch 'latest' into DOCS-6799

valkyrie69 committed:
Kim edits

adrian-sv committed:
Added the four new tables

1. Changes in operations parameters
2. Changes in operations metadata
3. New operations
4. AQuA API changes

valkyrie69 committed:
Merge pull request #544 from adrian-sv/patch-1

Added the four new tables

valkyrie69 committed:
minor edits

laszloczol committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Merge pull request #545 from laszloczol/patch-7

Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Reorg

valkyrie69 committed:
table formatting

valkyrie69 committed:
Minor edits

laszloczol committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Merge pull request #547 from laszloczol/patch-8

Update zuora-connector-upgrade-migrate.adoc

laszloczol committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Merge pull request #555 from laszloczol/patch-9

Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
minor edits

laszloczol committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Merge pull request #559 from laszloczol/patch-10

Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Update zuora-connector-upgrade-migrate.adoc

valkyrie69 committed:
Formatting tables

valkyrie69 committed:
Added the 4.2 reference to the nav

valkyrie69 committed:
Zuora 5.0 Updates

valkyrie69 committed:
added missining image
